odoo/cli/i18n.py

```diff
@@ -0,0 +1,228 @@
+            ),
+        )
+
+        self.loadlang_parser.add_argument(
+            '-l', '--languages', dest='languages', nargs='+', metavar='LANG',
+            help="List of language codes to install")
+
+    def run(self, cmdargs):
+        try:
+            parsed_args = self.parser.parse_args(args=cmdargs)
+        except ValueError as e:
+            self.parser.print_help()
+            sys.exit(f'\n{e}\n')
+
+        config_args = ['-c', parsed_args.config] if parsed_args.config else []
+        config.parse_config(config_args, setup_logging=True)
+        if not parsed_args.db_name:
+            db_names = config['db_name']
+            if not db_names or len(db_names) > 1:
+                self.parser.error("Please provide a single database in the config file")
+            parsed_args.db_name = db_names[0]
+
+        match parsed_args.subcommand:
+            case 'import':
+                self._import(parsed_args)
+            case 'export':
+                self._export(parsed_args)
+            case 'loadlang':
+                self._loadlang(parsed_args)
+
+    def _get_languages(self, env, language_codes, active_test=True):
+        # We want to log invalid parameters
+        Lang = env['res.lang'].with_context(active_test=False)
+        languages = Lang.search([('iso_code', 'in', language_codes)])
+        if not_found_language_codes := set(language_codes) - set(languages.mapped("iso_code")):
+            _logger.warning("Ignoring not found languages: %s", ', '.join(not_found_language_codes))
+        if active_test:
+            if not_installed_languages := languages.filtered(lambda x: not x.active):
+                languages -= not_installed_languages
+                iso_code_str = ", ".join(not_installed_languages.mapped("iso_code"))
+                _logger.warning("Ignoring not installed languages: %s", iso_code_str)
+        return languages
+
+    def _import(self, parsed_args):
+        paths = {Path(filename) for filename in parsed_args.files}
+        if invalid_paths := {path for path in paths if (
+            not path.exists()
+            or not path.suffixes
+            or path.suffixes[-1] not in IMPORT_EXTENSIONS
+        )}:
+            _logger.warning("Ignoring invalid paths: %s", ', '.join(invalid_paths))
+            paths -= invalid_paths
+        if not paths:
+            self.import_parser.error("No valid path was provided")
+
+        with Registry.new(parsed_args.db_name).cursor() as cr:
+            env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+            translation_importer = odoo.tools.translate.TranslationImporter(cr)
+            language = self._get_languages(env, [parsed_args.language])
+            if not language:
+                self.import_parser.error("No valid language has been provided")
+            for path in paths:
+                with open(path, "rb") as infile:
+                    translation_importer.load(infile, path.suffixes[-1].lstrip('.'), language.code)
```

Comment:
```suggestion
                    translation_importer.load(infile, path.suffixes[-1].removeprefix('.'), language.code)
```
spreading the good word: http://docs.localhost/python/library/stdtypes.html#str.removeprefix

* `lstrip` removes has many `.` as there are at the beginning of the string
* `removeprefix` removes a single `.`